packages/react-scripts/scripts/init.js

```diff
@@ -76,6 +82,16 @@ module.exports = function(appPath, appName, verbose, originalDirectory) {
   }
   args.push('react', 'react-dom');
 
+  // Install additional template dependencies, if present
+  var templateDependenciesPath = path.join(appPath, '.template.dependencies.json');
```

Comment:
Is there any reason we're not using `package.json` for this? Having a special name is kinda weird and I'm worried that when people discover this feature, they'll actually start relying on it.